lib/src/validator/changelog.dart

```diff
@@ -31,7 +31,7 @@ class ChangelogValidator extends Validator {
       } on FormatException catch (_) {
         warnings.add("$changelog contains invalid UTF-8.\n"
             "This will cause it to be displayed incorrectly on "
-            "pub.dartlang.org.");
+            "pub.dev.");
```

Comment:
I wonder if we should be concerned about readability with this format - without the `https://` or a well-known TLD like `.org` I'm not sure if this will stand out as a URL...

Reply:
We could instead say "the Pub site (pub.dev)" or "the Pub site (https://pub.dev)".

Reply:
ack